app/experimenter/experiments/api/v5/types.py

```diff
@@ -222,6 +223,9 @@ def resolve_jexl_targeting_expression(self, info):
     def resolve_is_enrollment_paused(self, info):
         return self.is_paused_published
 
+    def resolve_is_enrollment_pause_pending(self, info):
+        return self.is_paused and not self.is_paused_published
+
```

Comment:
This combined with status=LIVE / status_next = LIVE should give us all the hints needed on the frontend

Reply:
Wait explain this.  Shouldn't you just be able to infer all this based on publish_status? Why do you need to check the difference between the pause and published pause state?

Reply:
Can't infer specifically that this is a pending change to end enrollment, and I wanted to be specific https://github.com/mozilla/experimenter/pull/5852#discussion_r670715048